docs/source/integrations/mobile/ios-fairplay-sdk.md

```diff
@@ -87,125 +86,115 @@ More information about ATS can be found in Apple’s TechNote:
-
-To cancel a download, call:
-
-	`assetDownloadTask.cancel()`
-
-##### Legacy methods:
-
-These are the originally provided methods which are for online playback use only, and are provided for ease of upgrade for legacy users. These methods now also require an asset ID in the initialiser to be compatible with the new framework.
-
-- Legacy method 1: Pass in a reference to your AVPlayer instance, the token to be used, and the asset ID when retrieving a license.
-
-```swift
-let player = AVPlayer(url: assetURL!)
-self.drm = vudrmFairPlay(player: player, token: token, contentID: contentID)
-```
-
--	Legacy method 2: Pass in a reference to your stream URL and the token to be used when retrieving a license. An AVAsset will be created by VUDRMFairPlay and this can be used to create your player.
-
-```swift
-self.drm = vudrmFairPlay(url:assetURL! as NSURL, token: token, contentID: contentID)
-let playerItem = AVPlayerItem(asset: (drm?.asset)!)
-let player = AVPlayer(playerItem: playerItem)
-```
-
-Finally, present your player:
-
-```swift
-self.player = player
-let playerController = AVPlayerViewController() 
-playerController.delegate = self as? AVPlayerViewControllerDelegate
-self.present(playerController, animated: true, completion: nil)
-playerController.view.frame = self.view.frame
-playerController.player = self.player player.play()
-```
+To initialise your assets DRM instances, your application should create a DRM instance for each entry in the `Streams.plist` file (or comparable configuration) using a `ContentKeyManager`. How to do this is demonstrated in our demo application. The `ContentKeyManager` attaches instances of vudrmFairPlay to an `AssetResourceLoaderDelegate` for each asset using the asset URL, content ID and valid token. The content ID should be unique to each asset, as it is used to create a path to, and identify, offline assets and their associated content keys.
 
 ##### DEMO
 
-With the VUDRMFairPlay framework installed, the example demo application can be initialised either via source code in the IDE, or using field entry on device.
-
-- To initialise via code in IDE, before running, simply replace the empty ViewController.swift values with your own valid values.
-
-```swift
-private var assetURL = URL(string: "")
-private var token = ""
-private var contentID = ""
-```
-
-- To initialise via view controller on device, populate the URL, Token, and Asset Name fields with valid values. For ease of use, pasting into the URL field in the form URL + space + Token will auto populate both fields correctly. At runtime, any values in the URL and Token fields will override any hard coded values.
+With the VUDRMFairPlay framework installed, the example demo application uses entries in the `Streams.plist` file to configure the DRM. You can edit the `Streams.plist` file values, and/or add your own valid values, ensuring the `content_key_id_list` entry is correct, and that your `token` is valid (see **Preparation** above).
```

Comment:
If "Preparation" is a heading from above you should be able to link to so people can just click "Preparation" to jump to the relevant section.

Reply:
Full menu will be pushed after this has been merged, as discussed.